src/cli.js

```diff
@@ -15,11 +15,57 @@ const logging = require("./logging.js");
 
 const SCHEMASTORE_CATALOG_URL =
   "https://www.schemastore.org/api/json/catalog.json";
+
+const SCHEMASTORE_CATALOG_SCHEMA_URL =
+  "https://www.schemastore.org/api/json/schema-catalog.json";
```

Comment:
```suggestion
  "https://json.schemastore.org/schema-catalog.json";
```

This should be https://json.schemastore.org/schema-catalog.json
https://www.schemastore.org/api/json/schema-catalog.json is an error

Reply:
You're right. I was making some tests and I forget to undo it before committing. Resolved by 7b3428af9bfd65407dfceebcb7e6bd12de6de600.